### ISSUE TITLE
Module not installable in Py3 environment

### ISSUE THREAD

**Issue by ViktorHaag** [markstory/sphinxcontrib-phpdomain#10]
Trying to newly install in a Python3 environment (Python3.6.1 on MacOS 10.12.4, installed via Homebrew), I get these errors:

```
~ :  pip3.6 install sphinxcontrib-phpdomain
Collecting sphinxcontrib-phpdomain
  Using cached sphinxcontrib-phpdomain-0.2.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/7l/x3rh0rcs58q189dkstnqtt9hygmg20/T/pip-build-p9yhahzl/sphinxcontrib-phpdomain/setup.py", line 5, in <module>
        requirements = open('./requirements.txt', 'r')
    FileNotFoundError: [Errno 2] No such file or directory: './requirements.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/7l/x3rh0rcs58q189dkstnqtt9hygmg20/T/pip-build-p9yhahzl/sphinxcontrib-phpdomain/
~ :  
```

The module installs fine under Python2, same host OS.

**Comment by ViktorHaag** [comment]
Addressed with PR #11

**Comment by ViktorHaag** [comment]
Crap. I can install from source using pip, and I can build an sdist using py3, but for some reason this is still an issue installing with pip under Py3:

```
~ :  pip3.6 install sphinxcontrib-phpdomain
Collecting sphinxcontrib-phpdomain
  Using cached sphinxcontrib-phpdomain-0.2.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/7l/x3rh0rcs58q189dkstnqtt9hygmg20/T/pip-build-pvv1icqz/sphinxcontrib-phpdomain/setup.py", line 6, in <module>
        'requirements.txt'), 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/7l/x3rh0rcs58q189dkstnqtt9hygmg20/T/pip-build-pvv1icqz/sphinxcontrib-phpdomain/requirements.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/7l/x3rh0rcs58q189dkstnqtt9hygmg20/T/pip-build-pvv1icqz/sphinxcontrib-phpdomain/
```

Looks like this issue needs re-opening. Sorry about that, @markstory -- I thought I had it resolved with my PR. 8(

**Comment by ViktorHaag** [comment]
The problem appears to be that when the distribution is built, it's not including `requirements.txt` at all in the tarball.

**Comment by markstory** [comment]
Ok. I can fix the missing file.

**Comment by ViktorHaag** [comment]
Added fix with PR #12 .

**Comment by markstory** [comment]
new tag up with latest fixes.
